### ISSUE TITLE
New Site Style

### ISSUE THREAD

**Issue by davidroberts63** [techlahoma/okcsharp-website#41]
Any objections to me giving the site a refresh?

I was thinking about one of these two (other suggestions are gladly welcome):
https://html5up.net/striped
https://html5up.net/read-only

**Comment by the-simian** [comment]
Those are both really cool

**Comment by matthewreily** [comment]
I agree on them both being cool and would be happy with either.  The first
one is my favorite by a slight margin.
